.Rprofile

```diff
@@ -0,0 +1,3 @@
+options(
+  warnPartialMatchArgs = FALSE
```

Comment:
this is possibly problematic as I think it prevents reading of the user Rprofile - what is this trying to achive?

Reply:
Oops, I pushed this by mistake. Thanks